third_party/xla_client/xrt_computation_client.cc

```diff
@@ -1083,7 +1083,16 @@ std::unique_ptr<xrt::XLAComputation> XrtComputationClient::CreateXrtComputation(
   if (devices.size() > 1) {
     auto device_assignment = config->mutable_device_assignment();
     auto computation_device = device_assignment->add_computation_devices();
-    for (int64_t i = 0; i < devices.size(); ++i) {
+    auto local_device = GetLocalDevices();
+    int64_t local_tpu;
```

Comment:
Initialize this to some invalid value, like -1. Then do a `XLA_CHECK` after the for loop to ensure it's assigned a valid value.